signature/Cargo.toml

```diff
@@ -19,7 +19,9 @@ tokio = { version = "1.0", features = [ "rt", "macros", "rt-multi-thread" ] }
 tonic = "0.5"
 prost = "0.8"
 strum_macros = "0.24.2"
+ocicrypt-rs = { git = "https://github.com/confidential-containers/ocicrypt-rs" }
+attestation_agent = { git = "https://github.com/confidential-containers/attestation-agent" }
```

Comment:
Can we add a commit rev here? This can avoid potential problems caused by API changes in `AA`. The same as `ocicrypt-rs`.

Reply:
Sure. It is meaningful.